### PR TITLE
Set BasicCardResponse defaults in IDL, not in prose

### PR DESCRIPTION
**Pull request by marcoscaceres**
We are currently setting things in prose that we should be setting in the WebIDL.

The following tasks have been completed:

 * [X] Confirmed there are no ReSpec errors/warnings.
 * [X] Added Web platform tests - current tests already assume this behavior. 

Implementation commitments:

 * [ ] [Chrome](https://crbug.com/945318)
 * [x] [Firefox](https://bugzilla.mozilla.org/show_bug.cgi?id=1538526)
 * [ ] Edge (public signal)

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/payment-method-basic-card/pull/73.html" title="Last updated on Mar 26, 2019, 12:41 AM UTC (41d8f9f)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/payment-method-basic-card/73/0046bf8...41d8f9f.html" title="Last updated on Mar 26, 2019, 12:41 AM UTC (41d8f9f)">Diff</a>

### CODE REVIEW

**Comment by marcoscaceres** [comment]
[Patched Firefox](https://bugzilla.mozilla.org/show_bug.cgi?id=1538526) - but will wait for feedback before landing.

**Comment by rsolomakhin** [comment]
Linked to https://crbug.com/945318 in the description

**Review by rsolomakhin — APPROVED** [review]

**Pull request merged**
